deliver/spec/app_screenshot_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'deliver/app_screenshot'
+require 'deliver/setup'
+
+describe Deliver::AppScreenshot do
+  before do
+    allow(FastImage).to receive(:size).and_return([2732, 2048])
+  end
+
+  describe "should resolve iPad Pro (12.9-inch) ambiguity if" do
```

Comment:
According to better RSpec, you can write specs in this way.

http://www.betterspecs.org/

```ruby
describe "#initialize" do
  context "when filename doesn't contain '(3rd generation)'" do
     it "returns iPad Pro(12.9-inch) 3rd generation" do
     end
  end

  context "when filename contains '(3rd generation)" do
     it "returns iPad Pro(12.9-inch)" do
     end
  end
end
```